mbofui/app/js/utils.js

```diff
@@ -91,12 +91,17 @@ var processDates = function(startTime, endTime) {
 
 // new event validation - either the dates are blank or wrong or the description is blank
 var validate = function(data) {
+  // data we want to validate
+  var toValidate = ['eventText', 'title', 'startTime', 'endTime', 'address'];
```

Comment:
Is there any other place we use the literals 'eventText', 'title', 'startTime', 'endTime', 'address' so we can make these constants instead of literal? Does it make sense to do so?